trie/zk_trie_impl.go

```diff
@@ -353,12 +353,14 @@ func (mt *ZkTrieImpl) calcCommitment(rootKey *zkt.Hash, hashedDirtyNodes map[zkt
 func (mt *ZkTrieImpl) tryGet(nodeKey *zkt.Hash) (*Node, []*zkt.Hash, error) {
 
 	path := getPath(mt.maxLevels, nodeKey[:])
-	nextKey := mt.rootKey
+	var nextKey zkt.Hash
+	nextKey.Set(mt.rootKey)
```

Comment:
I wonder why here need deep copy?

Reply:
We need to call nextKey.Set in L389 and L392. If we don't deep copy here, those lines would change `mt.rootKey` unintentionally.